.github/workflows/ci.yml

```diff
@@ -104,3 +104,13 @@ jobs:
       - name: Check transpilation did not modify the python project
         run: |
           git diff --exit-code
+
+      - name: Check ontology capitalization (X-ray vs X-Ray)
+        run: |
+          FILES="./ontologies/esrfet/ESRFET.owl ./ontologies/esrffair/ESRFFAIR.owl"
+          if grep -nR "X-Ray" $FILES; then
```

Comment:
[minor] there is still an "x-ray", we could maybe check 
`grep -nRE "(X-Ray|x-ray)" $FILES | grep -vE 'IRI="#|<IRI>#'`

```
./ontologies/esrfet/ESRFET.owl:4673:                x-ray powder diffraction (technique) requires &apos;exactly&apos; powder</Literal>
```

but maybe a little overkill!